public/app/features/dashboard-scene/scene/NavToolbarActions.tsx

```diff
@@ -19,6 +21,8 @@ interface Props {
 export const NavToolbarActions = React.memo<Props>(({ dashboard }) => {
   const { actions = [], isEditing, viewPanelScene, isDirty, uid, meta, editview } = dashboard.useState();
   const toolbarActions = (actions ?? []).map((action) => <action.Component key={action.state.key} model={action} />);
+  const rightToolbarActions: JSX.Element[] = [];
+  const _legacyDashboardModelCompatibility = new DashboardModelCompatibilityWrapper(dashboard).asDashboardModel();
```

Comment:
instead of a creating a new wrapper here, I think it's better to use the one created in DashboardScene activation, and you assess that one simply by calling getDashboardSrv.getCurrent()